### ISSUE TITLE
What is the split ratio between the training set and test set of the VCTK data provided in your project?

### ISSUE THREAD

**Issue by jaxlinksync** [facebookresearch/loop#32]

**Comment by adampolyak** [comment]
7% of the data was used as test set.

**Comment by jaxlinksync** [comment]
Thank you @adampolyak
